launcher/ui/MainWindow.cpp

```diff
@@ -784,9 +775,6 @@ class MainWindow::Ui
         instanceToolBar->addSeparator();
 
         instanceToolBar->addAction(actionLaunchInstance);
-        instanceToolBar->addAction(actionLaunchInstanceOffline);
-
-        instanceToolBar->addSeparator();
```

Comment:
IMO it makes more sense to leave the separator in (even though there's only one action in that section now), since launching doesn't really fit with the stuff below (which all has to do with the instance's settings). Though this is pretty minor.